playbooks/nvidia-dgx-fw-update.yml

```diff
@@ -0,0 +1,14 @@
+---
+# This playbook updates the firwmware on DGX nodes
+# Refer to the README in roles/nvidia-dgx-firmware for more info
+
+- hosts: all
+  become: yes
+  connection: paramiko_ssh
```

Comment:
probably should remove this

Reply:
The new firmware container requires running with `-it` with the `update_fw` option. When using the default ssh connection plugin (set to `auto`), it complains about not having a tty. This was the only workaround that seemed to work.

Reply:
I can leave a comment explaining this in the playbook - would that help?